docker/launchTomcat.sh

```diff
@@ -74,8 +75,8 @@ fi
 
 #---------   CATALINA_OPTS must be exported for catalina.sh to pick them up
 export CATALINA_OPTS="\
-  -XX:InitialRAMPercentage=${INIT_RAM_PERCENT} \
-  -XX:MaxRAMPercentage=${MAX_RAM_PERCENT} \
+  -XX:InitialRAMPercentage=${INIT_RAM_PERCENT:-10} \
+  -XX:MaxRAMPercentage=${MAX_RAM_PERCENT:-90} \
```

Comment:
I thought you wanted to make this like 97 and use almost all of the memory.

Reply:
You're right.  I should go ahead with the change.